isis/src/base/objs/SurfacePoint/SurfacePoint.cpp

```diff
@@ -1038,18 +999,18 @@ namespace Isis {
       }
 
   /**
-   * Computes and returns the distance between two surface points. This does
-   *   not currently support ellipsoids and so any attempt with points with
-   *   planetary radii will fail. The average of the local radii will be
-   *   used.
+   * Computes and returns the distance between two surface points.  The average of 
+   * the local radii will be used.
    */
   Distance SurfacePoint::GetDistanceToPoint(const SurfacePoint &other) const {
-    if(p_majorAxis || p_minorAxis || p_polarAxis ||
-       other.p_majorAxis || other.p_minorAxis || other.p_polarAxis) {
-      IString msg = "SurfacePoint::GetDistanceToPoint not yet implemented for "
-          "ellipsoids";
-      throw IException(IException::Programmer, msg, _FILEINFO_);
-    }
+    // TODO What do we do for the check? We no longer have the ability to check.
```

Comment:
I think we just have to accept that this is the best we can do without the actual surface shape. Getting the true distance between points on a complex shape is probably too complex for what this is used for.